python/tvm/relax/transform/legalize_ops.py

```diff
@@ -774,7 +774,8 @@ def transform(self) -> IRModule:
                     if not isinstance(func, Function):
                         continue
                     updated_func = self.visit_expr(func)
-                    updated_func = remove_all_unused(updated_func)
+                    # removing to keep the match-cast statements
+                    # updated_func = remove_all_unused(updated_func)
```

Comment:
We should mention that this is a temporary solution and add TODO.

Reply:
Yes, agreed!

Reply:
Btw this has been fixed in other unity branches so I think won't be a program after rebase.

Reply:
Cool. Would you add a link for the fix to be clear for the person who will do the rebase?

Reply:
Sure.